listenbrainz/db/dump.py

```diff
@@ -153,7 +153,7 @@
 }
 
 
-def dump_postgres_db(location, dump_time=datetime.today(), threads=None):
+def dump_postgres_db(location, dump_time=datetime.today(), threads=DUMP_DEFAULT_THREAD_COUNT):
```

Comment:
Tests were failing because threads=None, caused xz to complain. Not sure why this worked earlier. pxz probably ignored invalid values of -T. Looking at docs, this is max number of threads to use for 
 compression/decompression. Some other dump methods use DUMP_DEFAULT_THREAD_COUNT for this so using it here as well.